02_bubble/shuutech/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+)
+
+func bubble(s []int) []int {
```

Comment:
I believe your code will change the passed in slice.

The requirement is to return a sorted copy, not amend the original slice

Reply:
i have fixed this, please review